WooCommerce/Classes/ViewRelated/Themes/ProfilerThemesPickerView.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftUI
+import Kingfisher
+
+struct ProfilerThemesPickerView: View {
+    /// Scale of the view based on accessibility changes
+    @ScaledMetric private var scale: CGFloat = 1.0
+
+    var body: some View {
+
+        ScrollView {
+            VStack(alignment: .leading) {
+                Spacer()
+                    .frame(height: Layout.contentVerticalSpacing)
+
+                Text(Localization.chooseThemeHeading)
+                    .bold()
+                    .largeTitleStyle()
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Spacer()
+                    .frame(height: Layout.contentPadding)
+
+                Text(Localization.chooseThemeSubtitle)
+                    .subheadlineStyle()
+                    .padding(.horizontal, Layout.contentPadding)
+
+                Spacer()
+                    .frame(height: Layout.contentVerticalSpacing)
+
+                ThemesCarouselView(
+                    lastMessageHeading: Localization.lastMessageHeading,
+                    lastMessageContent: Localization.lastMessageContent
+                )
+
+                Spacer()
```

Comment:
Since we want the carousel to be in the center of the remaining area, how about adding just flexible sizes for the `Spacer`'s:

```suggestion
                Spacer()

                ThemesCarouselView(
                    lastMessageHeading: Localization.lastMessageHeading,
                    lastMessageContent: Localization.lastMessageContent
                )

                Spacer()
```

Reply:
Another case where I followed the design too literally. This looks better! 👍🏼